symfony/flex/1.0/.env

```diff
@@ -1,5 +1,13 @@
-# This file defines all environment variables that the application needs.
-# DO NOT DEFINE PRODUCTION SECRETS IN THIS FILE.
-# Use ".env.local" for local overrides during development.
-# Use real environment variables when deploying to production.
+# In all environments, the following files are loaded
+# if they exist:
```

Comment:
Can you can talk about var precedence?

Reply:
No. Like variable definer in `.env` are overrided with var in `.env.local` etc.

Reply:
updated